vega_sim/scenario/common/agents.py

```diff
@@ -1315,6 +1303,12 @@ def _move_side(
         submissions = []
         cancellations = []
 
+        expires_at = (
+            int((self.vega.get_blockchain_time() + 5 * self.step_length_seconds) * 1e09)
```

Comment:
```suggestion
            int((self.vega.get_blockchain_time() + 5 * self.step_length_seconds) * 1e9)
```

Reply:
`* 5 ` was a safety margin, sometimes the environment may take longer to step then an agents `step_length_seconds` e.g. has to switch data-node or a query is slow). I'll make this configurable.

I named it `step_length_seconds` as I wanted it to be clear it was the same values as the `step_length_seconds` of the environment. Not sure how I would default the value.

Maybe it should be an agent wide property set in the `initialise` method?

Reply:
Could we not achieve the same thing slightly more clearly, and allow the agent itself to be more flexible, with having the parameter `order_validity_seconds` and setting that to `step_length_seconds * 5` when that's what we want?

Reply:
Ah nice yeah this is much clearer, will make that chage 👍